demos/CMakeLists.txt

```
@@ -20,6 +20,9 @@ libhal_build_demos(
   DEMOS
   input
   nunchuck
+  ch9329_nunchuck_mouse
```

```suggestion
  # ch9329_nunchuck_mouse
```

Lets comment this out until we release the libhal-input package with these libraries.